src/main/webapp/assets/modules/common/progressFilter.js

```diff
@@ -14,14 +14,14 @@ techlooper.filter("progress", function (jsonValue) {
         //if start date <= current date <= register date
         var registrationDate = moment(contestDetail.registrationDateTime, jsonValue.dateFormat);
         contestDetail.progress = jsonValue.status.registration;
-        if (moment().isBetween(startDate, registrationDate) || moment().isSame(startDate) || moment().isSame(registrationDate)) {
+        if (moment().isBetween(startDate, registrationDate) || moment().isSame(startDate, 'day') || moment().isSame(registrationDate, 'day')) {
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'moment' is not defined.